### PR TITLE
Sqlserver readme documentation fixes

### PR DESCRIPTION
**Pull request by bharadwaj-aditya**
PR to correct DB names and data type definitions in the readme.

- [x] Tests pass
- [x] Appropriate changes to README are included in PR

### CODE REVIEW

**Review by manitgupta — APPROVED** [review]
LGTM

File: sources/sqlserver/README.md [hunk 1]
```diff
@@ -2,16 +2,16 @@
 
 HarbourBridge is a stand-alone open source tool for Cloud Spanner evaluation,
 using data from an existing SQL Server database. This README provides
-details of the tool's SQLserver capabilities. For general HarbourBridge information
+details of the tool's SQL server capabilities. For general HarbourBridge information
```
Comment (Deep1998) [inline sources/sqlserver/README.md:5]: nit: SQL Server

File: sources/sqlserver/README.md [hunk 2]
```diff
@@ -2,16 +2,16 @@
...
 
 We currently do not support dump file mode for SQL Server. The only way to use HarbourBridge with SQL Server is connecting directly.
 
 Note that either _'sqlserver'_ or _'mssql'_ can be used as an identifier with the flag `-source` in the command line.
 
-## Example SQLServer Usage
+## Example SQL Server Usage
 
-HarbourBridge can be run directly on a sqlserver database (via go's database/sql package).
+HarbourBridge can be run directly on a SQL server database (via go's database/sql package).
```
Comment (Deep1998) [inline sources/sqlserver/README.md:14]: nit: SQL Server

File: sources/sqlserver/README.md [hunk 3]
```diff
@@ -30,7 +30,7 @@ harbourbridge schema -source=sqlserver -source-profile="host=<>,port=<>,user=<>,
 ```
 
 Parameters `port` and `password` are optional. Port (`port`) defaults to `1433`
-for SQLserver source. Password can be provided at the password prompt.
+for SQL server source. Password can be provided at the password prompt.
```
Comment (Deep1998) [inline sources/sqlserver/README.md:33]: nit: SQL Server

**Review by Deep1998 — APPROVED** [review]
LGTM, just a few nits for consistency

**Pull request merged**